package.json

```diff
@@ -9,7 +9,7 @@
     "npm": "^8.5.5"
   },
   "devDependencies": {
-    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git",
+    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#bugfix/fp-1542-figure-image-margin",
```

Comment:
**Warning.** Do not commit the branch URL to this file. Re install/lock after merge of https://github.com/TACC/Core-Styles/pull/25.